scala/integration/intellij-bsp/src/org/jetbrains/plugins/scala/bsp/flow/open/ScalaBspDetailsConnectionGenerator.scala

```diff
@@ -28,9 +29,26 @@ class ScalaBspDetailsConnectionGenerator extends BspConnectionDetailsGeneratorEx
     getChild(projectPath, List(".bsp", "sbt.json").asJava)
   }
 
-  override def calculateImportWizardSteps(path: Path, observableMutableProperty: ObservableMutableProperty[ConnectionFileOrNewConnection]): util.List[ImportProjectWizardStep] = ???
+  override def calculateImportWizardSteps(path: Path, observableMutableProperty: ObservableMutableProperty[ConnectionFileOrNewConnection]): util.List[ImportProjectWizardStep] = List.empty.asJava
```

Comment:
nit: use JavaList here as well (unless there's a different convention in intellij-scala)